lib/recma-stringify.js

```diff
@@ -1,12 +1,31 @@
+// @ts-ignore baseGenerator is not yet exported by astring typings
 import {baseGenerator, generate} from 'astring'
 import stringifyEntities from 'stringify-entities/light.js'
 
-// A small wrapper around `astring` to add support for serializing JSX.
+/**
+ * @typedef {import("unist").Node} Node
+ * @typedef {import("vfile").VFile} VFile
+ *
+ * @typedef RecmaStringifyOptions
+ * @property {*} [SourceMapGenerator]
```

Comment:
does this specifically have to be the source map generator library?
Or are there others that could be used?

Reply:
yes, specifically from `source-map`